src/components/Buttons.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./Button/BaseButton";
+export * from "./Button/InputButton";
+export * from "./Button/LinkButton";
+export * from "./Button/Button";
```

Comment:
I think let's put all of these directly in `./Button/index.ts`, so a user can import any of them from `lbh-frontend/../components/Button`.